app/src/test/java/com/github/se/cyrcle/model/user/UserViewModelTest.kt

```diff
@@ -68,14 +68,15 @@ class UserViewModelTest {
     // Check if the user was fetched from the repository
     verify(userRepository).getUserById(eq("user1"), any(), any())
   }
-  // Check that the user returned is the correct one
-  // and that onSuccess is called.
+
+  @Suppress("UNCHECKED_CAST")
```

Comment:
Why is this annotation necessary?

Reply:
Because the cast of the callbacks create a warning while everything is under control 😉